src/bptl/conf/base.py

```diff
@@ -493,3 +493,6 @@
 MAX_DOCUMENT_SIZE = os.getenv("MAX_DOCUMENT_SIZE", 2621440)
 # The size of the document chunks to load in memory
 CHUNK_SIZE = os.getenv("CHUNK_SIZE", 4096)
+
+# Xential ticket settings
+XENTIAL_URL_TOKEN_TIMEOUT_DAYS = os.getenv("XENTIAL_URL_TOKEN_TIMEOUT_DAYS", default=7)
```

Comment:
```suggestion
XENTIAL_URL_TOKEN_TIMEOUT_DAYS = config("XENTIAL_URL_TOKEN_TIMEOUT_DAYS", default=7)
```
pretty much everywhere `os.getenv` should be replaced with `config`